buildSrc/build.gradle

```diff
@@ -25,7 +25,7 @@ repositories {
 }
 
 dependencies {
-  testRuntime('org.apache.geode:geode-junit:1.3.0') {
+  testRuntimeOnly('org.apache.geode:geode-junit:1.3.0') {
     exclude group: 'org.apache.logging.log4j'
```

Comment:
Do we end up excluding log4j all over? Maybe we should make it `compileOnly` in `geode-junit`?